### DISCUSSION TITLE
Serverproblems?

### DISCUSSION THREAD

**Discussion by Amokd0c** [alexkn/node-red-contrib-homeconnect#89] [Q&A]
Hi!
I have no connection from node red to home connect. Did someone also get this notifcation also? 
"not connected" is shown under the node.
IO Broker is still working.

Thank you!

**Comment by Cafun** [comment]
Mine are still connected.
How do you deploy Node-Red when you editted something?
Mine is set on "Modified Nodes". When My Home Connected is showing "not connected" I set deploy mode to "Full" and place one node a bit down or up and then deploy. It makes it connect again.

**Comment by Amokd0c** [comment]
Ok thanks for your reply! I know this procedure. Node Red is running as an instance in IO Broker. I get this message in the log.

15 Apr 21:52:53 - [error] [home-connect-auth:Node Red HomeConnect WM] refreshTokens failed: { "error": "429", "error_description": "The rate limit for refresh token requests was reached. Requests are blocked during the remaining period of 40159 seconds."}

I guess I only have to wait! :)
